scripts/get-changed-services/index.ts

```diff
@@ -0,0 +1,146 @@
+import { Octokit } from '@octokit/action'
+import * as core from '@actions/core'
+
+import { exec } from 'child_process'
+import util from 'util'
+/*
+import fs from 'fs'
+// just for testing locally now
+function readToken(path = '../../../../../access_token.txt') {
+    return fs.readFileSync(path).toString().trim()
+}
+
+process.env.GITHUB_ACTION = 'true'
+process.env.GITHUB_TOKEN = readToken()
+*/
+
+// for now keep this list hardcoded
```

Comment:
urgh. Would you add this section to our README plz

"How to add a new service"
- make a folder in `/services`. 
- add a `package.json`,  `serverless.yml` and a `README.md` inside
- if you plan to write js/ts code, make a a `src` folder first. This is a Node standard.
- to add your services to our deploy flow - Add a step to `deploy-app-to- env` and add your service to list in `get-changed-services` scripts

Reply:
Sounds good, I'll add a section like this to the README. 

I can also give getting this list of services programmatically another go, I have a func in this action that pulls it from Lerna, but hardcoded to get myself going faster.

Reply:
yeah thats fine just want to start documenting stuff like this.